extensions/amp-ad-exit/0.1/config.js

```diff
@@ -121,6 +134,20 @@ function assertTarget(name, target, config) {
       user().assert(
           pattern.test(variable), '\'%s\' must match the pattern \'%s\'',
           variable, pattern);
+      const vendor = variable.substr(1);
```

Comment:
Please double-check that I'm understanding your logic correctly - I'm a bit confused about whether a custom variable *has* to identify a vendor, or if you can have a non-vendor custom variable for some reason.